client/frame-runner.js

```diff
@@ -71,7 +69,7 @@ document.addEventListener('DOMContentLoaded', function() {
             // the function could expect a callback
             // or it could return a promise/observable
             // or it could still be sync
-            if (test.length < 1) {
+            if (test.length === 1) {
```

Comment:
I'm looking at your code to adapt my dummy test framework on Hyperdev to the new test format using `getUserInput()`. I think I found a potential bug. Is checking for `test.length === 1` correct. In this way, if we pass a test string containing a function with 0 arguments, it will be handled as a node callback, by the following `else` block. What do you think ?